test/ussd_community.test.js

```diff
@@ -0,0 +1,1908 @@
+                                msisdn: '+2348080020002',
+                                identity: 'identity-uuid-1002',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_IdentityStoreDynamic().get_identity({
+                                msisdn: '+2348080020002',
+                                identity: 'identity-uuid-1002',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_MessageSenderDynamic().create_outbound({
+                                identity: 'identity-uuid-1002',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_IdentityStoreDynamic().patch_identity({
+                                msisdn: '+2348080020002',
+                                identity: 'identity-uuid-1002',
+                                extra_details: {
+                                    "dialback_sent": true
+                                }
+                            }));
+                    })
+                    .setup.user.addr('08080020002')
+                    .inputs(
+                        {session_event: 'new'}  // dial in
+                        , '12345'  // state_auth_code - corp code
+                        , {session_event: 'close'}
+                    )
+                    .check(function(api) {
+                        utils.check_fixtures_used(api, [0, 1, 2, 3, 4]);
+                    })
+                    .run();
+            });
+            it("should not send a dialback sms on second timeout", function() {
+                return tester
+                    .setup(function(api) {
+                        api.http.fixtures.add(
+                            fixtures_IdentityStoreDynamic().identity_search({
+                                msisdn: '+2348080040004',
+                                search_value: '12345',
+                                identity: 'identity-uuid-1004',
+                                search_param: 'details__corp_code',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_IdentityStoreDynamic().identity_search({
+                                msisdn: '+2348080040004',
+                                identity: 'identity-uuid-1004',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_IdentityStoreDynamic().get_identity({
+                                msisdn: '+2348080040004',
+                                identity: 'identity-uuid-1004',
+                            }));
+
+                        api.http.fixtures.add(
+                            fixtures_MessageSenderDynamic().create_outbound({
+                                identity: 'identity-uuid-1004',
+                            }));
```

Comment:
Maybe I'm a little confused, but it doesn't seem like this test is testing that the SMS isn't being sent, especially since it has the outbound creation fixture.

Reply:
It's the dialback sms that is being sent here because of the `{session_event: 'close'}` event.